lua/autorun/nl_library.lua

```diff
@@ -0,0 +1,609 @@
+			return
+		end
+		timer.Create("NLLSleepIn"..id, timebefore, 1, function()
+			NLL.Ragdoll(ply, pushdir,stuntype)
+			ply.nllismuted = true
+			timer.Create("NLLUnragdoll"..id, timedur, 1, function()
+				NLL.UnRagdoll(ply)
+				ply.nllismuted = false
+			end)
+		end)
+		ply.DrowsyDuration = timebefore
+		net.Start("nllinfopayload")
+			net.WriteString("drowsy")
+			net.WriteTable({["drowsy"] = timebefore})
+		net.Send(ply)
+		return
+	end
+	if stuntype == "STN" or stuntype == "TZE" then -- Taze shakes more violently than Stun
+		if not ply.nllstnpr then ply.nllstnpr = 0 end
+		ply.nllstnpr = ply.nllstnpr + timebefore -- Stun Damage
+		if ply.nllstnpr < 100 then // Not above 100 Stun Damage, don't do anything.
+			if timer.Exists("NLLSTNHeal"..id) then timer.Remove("NLLSTNHeal"..id) end
+			timer.Create("NLLSTNHeal"..id,15,1,function()
+				ply.nllstnpr = 0 -- They haven't been hit by stun for 15 seconds, heal that stun damage
+			end)
+			return
+		end
+		timer.Create("NLLSleepIn"..id,0,1,function()
+			ply.nllstnpr = 0
+			NLL.Ragdoll(ply, pushdir,stuntype)
+			ply.nllismuted = true
+			timer.Create("NLLUnragdoll"..id, timedur, 1, function()
+				NLL.UnRagdoll(ply)
+				ply.nllismuted = false
+			end)
+		end)
+		return
+	end
+	if stuntype == "---" then -- "Surrender". Supposed to allow self ragdolling, unused for now.
+		NLL.Ragdoll(ply, pushdir,"---")
+		ply.nllismuted = false -- You're still awake, you shouldn't be muted while surrendering.
+		timer.Create("NLLUnragdoll"..id, timedur, 1, function()
+			NLL.UnRagdoll(ply)
+			ply.nllismuted = false
+		end)
+		return
+	end
+	if stuntype == "PLV" then -- Pulverize, bone anihilation method
+		NLL.Ragdoll(ply, pushdir,"PLV")
+		ply.nllismuted = true
+		timer.Create("NLLUnragdoll"..id, timedur, 1, function()
+			NLL.UnRagdoll(ply)
+			ply.nllismuted = false
+		end)
+		return
+	end
+end
+else -- clientside code
+
+local condcolors = { -- Will add the ability to customize condcolors later
+	["ZZZ"] = {Color(150,150,255),"sleeping","wake up"},
+	["STN"] = {Color(255,150,150),"stunned"},
+	["KSR"] = {Color(255,0,255),"a floorkisser"},
+	["PLV"] = {Color(255,0,0),"suffering","make it stop"},
+	["---"] = {Color(150,150,150),"surrendering","make at ease"}
+}
+
+function NLL_BuddhaKillUpdate(ply,cmd,args)
+	if args[1] ~= "0" then
+		net.Start("nllupdatebuddha")
+		net.WriteInt(1,4)
+		net.SendToServer()
+		return
+	end
+	net.Start("nllupdatebuddha")
+	net.WriteInt(0,4)
+	net.SendToServer()
+	return
+end
+
+local NLL_COVERT_BUDDHA = CreateClientConVar("ttt_nll_covertbuddha", "0", true, true, "If nonzero, covertly killing will drop your target's hp to 1",0,1)
+
+net.Receive("nllinfopayload", function()
+	local payloadtype = net.ReadString()
+	local payload = net.ReadTable()
+	if string.find( payloadtype, "health", 1, false ) then
+		payload["subject"].newhp = payload["health"]
+	end
+	if string.find( payloadtype, "cond", 1, false ) then
+		payload["subject"].condition = payload["condition"]
```

Comment:
Same deal with using two different existing systems. Decide on either `ply.condition` or `NLL.ConditionStorage`.

Reply:
This is the clientside code, which does not need to loop through each conditioned player. Since I am not going to be using a table, I won't be defining a table.

Reply:
Then make the table shared? If anything, polluting the player table is worse than having your own stored which is already setup and easy to change.
Any reasonable maintainer would expect the same vars to be stored in the same table clientside and not elsewhere with a different system to achieve the same thing.
This also renders `NLL.GetCondtion` completely pointless as it's unused and never will be used by this logic.

Reply:
Ultimately it doesn't matter on the grand scheme of things, but it is unclean/bad and will make maintaining more bothersome so I will keep it resolved for now.